python/sdk/prelude_sdk/controllers/build_controller.py

```diff
@@ -12,13 +12,15 @@ def __init__(self, account):
         self.account = account
 
     @verify_credentials
-    def create_test(self, name, unit, technique=None, test_id=None):
-        """ Create or update a test """
+    def create_test(self, name=None, unit=None, technique=None, source_test_id=None, test_id=None):
```

Comment:
i'd either split into two methods to match CLI or change the logic to look more similar to the BE to clarify what params are required when. 
ie
```
if source_test_id:
  body = dict(source_test_id = source_test_id)
else:
  body = dict(name=name, unit=unit)
  if technique: blah
  if test_id: blah

res = session.post()
```